pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManager.java

```diff
@@ -613,7 +613,7 @@ private boolean processStreamEvents(MessageBatch messagesAndOffsets, long idlePi
         _segmentLogger.debug("empty batch received - sleeping for {}ms", idlePipeSleepTimeMillis);
       }
       // Record Pinot ingestion delay as zero since we are up-to-date and no new events
-      _realtimeTableDataManager.updateIngestionDelay(0, System.currentTimeMillis(), _partitionGroupId);
+      setIngestionDelayToZero();
```

Comment:
Other than adding an extra method call, is there any change here?

Instead, if we call `updateIngestionDelay(0, currentTime, partiongroupId`,  can we not handle all the ingstion delay specific logic inside that method?

Reply:
So are you suggesting we change the initial implementation of updateIngestion delay? I guess I can do that but it will then look pretty close to what we have initially: a method that passes a number of arguments.

Reply:
Discussed offline, thanks for the explanation. Your original call to realtimeTableDatamanager is good. Let us revert to that.

Reply:
We resolved this offline, we will keep things as they were.